.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -9,6 +9,19 @@ body:
         Found a bug? Please fill out the sections below. 👍
         Thanks for taking the time to fill out this bug report!
         For backend issues, use [lemmy](https://github.com/LemmyNet/lemmy)
+  - type: checkboxes
+    attributes:
+      label: Requirements
+      description: Before you create a bug report please do the following.
+      options:
+        - label: Did you check to see if this issue already exists?
+          required: true
+        - label: Is this only a single bug. Do not put multiple bugs in one issue.
```

Comment:
Question mark after the first sentence

Reply:
Fixed.
PS in the future recommend using suggested changes. https://haacked.com/archive/2019/06/03/suggested-changes/